packages/dapp-high-roller/src/components/app-wager/app-wager.tsx

```diff
@@ -1,12 +1,13 @@
-declare var commonTypes;
+// declare var commonTypes;
 declare var ethers;
 
 import { Component, Prop, State } from "@stencil/core";
 import { RouterHistory } from "@stencil/router";
 
 import CounterfactualTunnel from "../../data/counterfactual";
 
-const { AssetType } = commonTypes;
+// FIXME: Figure out how to import @counterfactual-types
```

Comment:
Why doesn't this work ATM?

Reply:
General import issue with stencil

https://github.com/counterfactual/monorepo/issues/362